docs/reduction-in-locks.md

```diff
@@ -0,0 +1,85 @@
+# Reduced backup lock time
+
+--8<--- "pro-build-announcement.md"
+
+!!! important
+
+    The `--lock-ddl=REDUCED` option is a [tech preview](./glossary.md#tech-preview). Before using this option in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+[Percona XtraBackup 8.4.0-2](./release-notes/8.4.0-2.md) adds the [`--lock-ddl=REDUCED`](./xtrabackup-option-reference.md#lock-ddl) option to reduce the time the server remains locked by `xtrabackup` during full and incremental backups. Now, you can execute `Data Definition Language` (DDL) operations while the backup is in progress.
+
+### Benefits
+
+The `--lock-ddl=REDUCED` option features are as follows:
+
+* Acquire and release the backup lock quickly: The backup process minimizes the time it holds the backup lock, allowing for concurrent DDL operations.
+* Track changes with redo logs: Redo logs record all file-level changes, ensuring data consistency during the backup process.
+* Handle DDL operations: The backup process generates metadata files to account for any DDL operations that occur while the backup is in progress.
+* Ensure consistency: The `--prepare` step processes generated metadata files and uses redo and undo logs to create a consistent database state.
+
+### Limitations
+
+* Certain DDL operations consume significant resources, and xtrabackup simultaneously requires I/O to copy and read files. This requirement can increase resource demand from DDL operations during the backup process.
```

Comment:
Also, it would be nice to mention in this section that while backups are being taken with REDUCED lock, we should not rotate the master key. Below DDL is prohibited while backups are taken with lock-ddl=REDUCED
ALTER INSTANCE ROTATE INNODB MASTER KEY

Reply:
added